### PR TITLE
added Github action that builds & run tests with simulator

### PR DESCRIPTION
**Pull request by Tapanila**
Solves #8

### CODE REVIEW

File: .github/workflows/ios.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    name: Build and Test FlipperZero(ios) scheme using iPhone 11 simulator
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Xcode build
+        run: |
+          sudo xcode-select --switch /Applications/Xcode_12.app
+          xcodebuild -sdk iphoneos -workspace ./FlipperZero/FlipperZero.xcodeproj/project.xcworkspace -scheme 'FlipperZero(iOS)'  clean archive -archivePath ./output/build/archive/FlipperZeroiOS.xcarchive CODE_SIGNING_ALLOWED=NO
```
Comment (evnik) [inline .github/workflows/ios.yml:20]: I think right now we don't need to archive on each PR update.
Later we might add archiving and uploading of the package on each push to `master`.

File: .github/workflows/ios.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - name: Test
+        run: |
+          xcodebuild -sdk iphonesimulator -workspace ./FlipperZero/FlipperZero.xcodeproj/project.xcworkspace -scheme 'FlipperZero(iOS)' test DSTROOT=./output/test/build.dst OBJROOT=./output/test/build.obj SYMROOT=./output/test/build.sym SHARED_PRECOMPS_DIR=./output/test/build.pch -destination platform='iOS Simulator,name=iPhone 11'
```
Comment (evnik) [inline .github/workflows/ios.yml:23]: I guess output of `xcodebuild` might be too verbose. Could you please add `xcpretty`?

**Review by evnik — COMMENTED** [review]
Thank you for your contribution! 👍 
Overall it looks good, however I have couple of comments

File: .github/workflows/ios.yml (hunk 1 above)
Comment (Tapanila) [reply .github/workflows/ios.yml]: Good point

File: .github/workflows/ios.yml (hunk 2 above)
Comment (Tapanila) [reply .github/workflows/ios.yml]: Haven't used that before. That made the output way better

**Review by evnik — APPROVED** [review]

**Pull request merged**